### PR TITLE
Set up cldr-runner to operate on Ubuntu 20.04 and RHEL 9 hosts

### PR DESCRIPTION
**Pull request by wmudge**
In order to run `ansible-navigator` and other `ansible-runner`-enabled applications on Ubuntu and RHEL hosts, rather than within a container via Docker.

### CODE REVIEW

**Review by jimright — CHANGES_REQUESTED** [review]
Hey @wmudge,

I've tested this by spinning up some EC2 instances for RHEL9 & Ubuntu (both 20.04 and 22.04) and running the scripts from the command line.
I also tested using these scripts as user data inputs. 

See my comments for some suggested changes.

As the script needs to be executed with elevated privileges (e.g. `sudo ...`), one thing that might also be worth considering is how to get the .bashrc prompt for the executing user. 
You could capture this using something like `login_user=${SUDO_USER:-$USER}` or by adding sudo on only commands that require it.

File: NAVIGATOR.md [hunk 1]
```diff
@@ -109,7 +109,12 @@ Arguably, this mode is the most complex, as it requires you to have installed al
 
 See the [Execution Environment configuration](https://ansible.readthedocs.io/projects/navigator/settings/#execution-environment) section for further details.
 
-This is a fairly common task then [running CI jobs](https://github.com/cloudera-labs/cloudera.cluster/blob/main/.github/workflows/validate_pr.yml) for individual collections. The [local development](README.md#local-development) instructions are designed to support this mode.
+This is a fairly common task then [running CI jobs](https://github.com/cloudera-labs/cloudera.cluster/blob/main/.github/workflows/validate_pr.yml) for individual collections. The [local development](README.md#local-development) instructions are designed to support this mode, and you can also bootstrap a host (such as a standalone VM) using following scripts:
+
+* `rhel9-init-base.sh` (RedHat 9)
+* `ubuntu20-init-base.sh` (Ubuntu 20.04 LTS)
```
[the remark this answers is not in the document]
Comment (wmudge) [reply NAVIGATOR.md:115]: Added a CASE statement based on the output of `lsb_release`.

File: rhel9-init-base.sh [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+yum-config-manager --add-repo https://rpm.releases.hashicorp.com/RHEL/hashicorp.repo
+yum -y install terraform
+
+# Use existing Python3.9 and pip
+python3 -m venv cdp-navigator
+source cdp-navigator/bin/activate
+pip install --upgrade pip
+pip install wheel
+pip install ansible-core~=2.12.10 ansible-navigator
+cat <<EOF >> ~/.bashrc
```
[the remark this answers is not in the document]
Comment (wmudge) [reply rhel9-init-base.sh]: Added to the global bashrc along with additional instructions for enabling the virtual environment

**Review by jimright — APPROVED** [review]
Looks good. Approved

**Pull request merged**